### ISSUE TITLE
Move under digitalnz org?

### ISSUE THREAD

**Issue by kinow** [fogonwater/pydnz#3]
Not really an issue, just curiosity if it would make sense to move the API under the DigitalNZ org in GitHub?

**Comment by fogonwater** [comment]
Yeah. It probably should go there. I started this as a personal project, but it now gets a reasonable amount of use. I'm on leave until September, but I will likely move it across once I return. Should be in a great state with lots of tests by then. ;)

**Comment by ajenhl** [comment]
Did this move happen, or has this project been abandoned?
